extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -253,54 +293,49 @@ public function doWikiForm() {
 					$info = $this->doUpdateTags( $wgRequest );
 					break;
 				case "findtags":
-					#we have 2 things that post from this page :(
-					if( $wgRequest->getVal('wpSearchTag') != null ) {
-						$this->mSearchTag = $wgRequest->getVal('wpSearchTag');
+					# we have 2 things that post from this page :(
+					if ( $wgRequest->getVal( 'wpSearchTag' ) != null ) {
+						$this->mSearchTag = $wgRequest->getVal( 'wpSearchTag' );
 						$info = $this->doSearchTags( $this->mSearchTag );
 					}
 
-						$this->mRemoveTag = $wgRequest->getVal('remove_tag');
-						$this->mRemoveTags = $wgRequest->getIntArray('remove_tag_id');
-					if( $this->mRemoveTag != null &&
+						$this->mRemoveTag = $wgRequest->getVal( 'remove_tag' );
+						$this->mRemoveTags = $wgRequest->getIntArray( 'remove_tag_id' );
+					if ( $this->mRemoveTag != null &&
 						$this->mRemoveTags != null ) {
 						$info = $this->doMultiRemoveTags();
 					}
 					break;
 				case "masstags":
-						$this->mMassTag = $wgRequest->getVal('wpMassTag');
-						$this->mMassTagWikis = $wgRequest->getVal('wfMassTagWikis');
+						$this->mMassTag = $wgRequest->getVal( 'wpMassTag' );
+						$this->mMassTagWikis = $wgRequest->getVal( 'wfMassTagWikis' );
 
-					if( $this->mMassTag != null && $this->mMassTagWikis != null ) {
+					if ( $this->mMassTag != null && $this->mMassTagWikis != null ) {
 						$info = $this->doMassTag();
-					}
-					else
-					{
-						if( $this->mMassTag == null )
-						{
-							$info = Wikia::errorbox('missing tag to apply to wikis');
+					} else {
+						if ( $this->mMassTag == null ) {
+							$info = Wikia::errorbox( 'missing tag to apply to wikis' );
 						}
-						if( $this->mMassTagWikis == null )
-						{
-							$info = Wikia::errorbox('missing wikis to apply tag to');
+						if ( $this->mMassTagWikis == null ) {
+							$info = Wikia::errorbox( 'missing wikis to apply tag to' );
 						}
 					}
 					break;
 				case "ezsharedupload":
-					if($wgRequest->getVal('ezsuSave') != null) {
+					if ( $wgRequest->getVal( 'ezsuSave' ) != null ) {
 						$info = $this->doSharedUploadEnable( $wgRequest );
 					}
 					break;
 			}
-			wfRunHooks('WikiFactory::onPostChangesApplied', array($this->mWiki->city_id));
-		}
-		else {
+			wfRunHooks( 'WikiFactory::onPostChangesApplied', [  $this->mWiki->city_id  ] );
+		} else {
 			/**
 			 * maybe some other action but with GET not POST
 			 */
 			switch( $this->mTab ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "case default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this "switch" statement with "if" statements to increase readability. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1301)